src/libraries/System.Net.Mail/src/System/Net/Mail/SmtpCommands.cs

```diff
@@ -641,6 +641,46 @@ internal static bool Send(SmtpConnection conn, string to, out string response)
         }
     }
 
+    internal static class QuitCommand
+    {
+        private static void CheckResponse(SmtpStatusCode statusCode, string serverResponse)
+        {
+            switch (statusCode)
+            {
+                case SmtpStatusCode.ServiceClosingTransmissionChannel:
+                    {
```

Comment:
I don't think we indent braces for case blocks like this.  Other code in repo leaves the braces aligned with 'case' and 'default' blocks.

Reply:
ok

Reply:
Ah understood @davidsh I indented in this way because it's the style in this specific file, would you like to fix also all others?

Reply:
ping @davidsh

Reply:
@scalablecory can you please take over the comment here as @davidsh is OOF for few more weeks. Thanks!

Reply:
Ok this code will be removed if we don't check response.